internal/provider/resource_saml_idp.go

```diff
@@ -23,6 +23,11 @@ func resourceWizSAMLIdP() *schema.Resource {
 				Description: "Internal identifier for the Saml Provider",
 				Computed:    true,
 			},
+			"issuer_url": {
+				Type:        schema.TypeString,
+				Description: "IdP issuer URL or entity ID. The value needs to match the IdP Issuer value, which *should* generally be a URL but can also be a URI or simple string. If unsure, set to same as login_url.",
+				Required:    true,
```

Comment:
This field is optional, per the GraphQL schema.

```
type SAMLIdentityProvider {
  allowManualRoleOverride: Boolean!
  certificate: String!
  domains: [String!]
  groupMapping: [SAMLGroupMapping!]
  id: ID!
  issuerURL: String
  loginURL: String!
  logoutURL: String!
  mergeGroupsMappingByRole: Boolean!
  name: String!
  useProviderManagedRoles: Boolean!
}
```

Reply:
Indeed it should be optional. This speaks to the bug that Wiz introduced when moving to Cognito which is, according to support, limited to this specific mutation in that all attributes once passed, need to be passed for the lifecycle. 

Assuming the value gets set at creation but unset at some stage, the API would return an error which is what was tested.

Reply:
Thanks for the clarification.  We can work around this with the golang struct.  The Resource schema can still make this optional.  Make sense?

Reply:
I created a saml idp without the issuerurl defined, but when reading the resource, it appears the issuerurl was set to the loginurl:

Create variables:

```
{
  "input": {
    "name": "test-bool",
    "loginURL": "https://example.com/idp/SSO.saml2",
    "logoutURL": "https://example.com/idp/SLO.saml2",
    "useProviderManagedRoles": true,
    "allowManualRoleOverride": true,
    "certificate": "-----BEGIN ..."
    "domains": [],
    "mergeGroupsMappingByRole": false
  }
}
```

Read response:

```
{
  "data": {
    "samlIdentityProvider": {
      "id": "test-bool",
      "name": "test-bool",
      "issuerURL": "https://example.com/idp/SSO.saml2",
      "loginURL": "https://example.com/idp/SSO.saml2",
      "logoutURL": "https://example.com/idp/SLO.saml2",
      "useProviderManagedRoles": true,
      "allowManualRoleOverride": true,
      "domains": null,
      "mergeGroupsMappingByRole": false,
      "groupMapping": []
    }
  }
}

Reply:
Indeed thats Wiz's default behaviour to set to loginURL if undefined. I tested when creating it in TF as undefined (when it was optional) the create would work, but the update would fail as it wasn't part of the original fields set at creation hence why this one also made required to work around.

Reply:
Will test this then if the read response updates state why I found above behaviour..\